src/openforms/forms/tests/factories.py

```diff
@@ -115,6 +135,14 @@ class Meta:
         model = "forms.FormRegistrationBackend"
 
 
+class FormAuthenticationBackendFactory(factory.django.DjangoModelFactory):
+    backend = factory.fuzzy.FuzzyChoice(authentication_plugins())
```

Comment:
do we need to *call* `authentication_plugins` or can you also provide it as a callback? Generally you want to avoid calling functions at the module level.